lib/hangman.rb

```diff
@@ -0,0 +1,54 @@
+class Word
+
+  attr_reader :incorrect_guesses, :correct_guesses, :user_guesses
+
+  def initialize(word)
+    @word = word
+    # @user_guesses = [] # not needed?
+    @incorrect_guesses = []
+    @correct_guesses = ["_"] * @word.length
+  end
+
+  def add_guess(letter)
+    if @incorrect_guesses.include?(letter) && @incorrect_guesses.include?(letter)
```

Comment:
Looks like this condition is duplicated `@incorrect_guesses.include?(letter)`

Reply:
Thank you! I've changed this to:
`@incorrect_guesses.include?(letter) || @correct_guesses.include?(letter)`